frontends/web/src/routes/new-settings/appearance.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+import { View, ViewContent } from '../../components/view/view';
+import { Header, Main } from '../../components/layout';
+import { DarkmodeToggleSetting } from './components/settingsItems/darkmodeToggleSetting';
+import { DefaultCurrencySetting } from './components/settingsItems/defaultCurrencySetting';
+import { DisplaySatsToggleSetting } from './components/settingsItems/displaySatsToggleSetting';
+import style from './appearance.module.css';
+import { LanguageDropdownSetting } from './components/settingsItems/languageDropdownSetting';
+import { ActiveCurrenciesDropdownSetting } from './components/settingsItems/activeCurrenciesDropdownSetting';
```

Comment:
I'd prefer all settings items that belong to the same settings category in one place. Maybe in

`/components/appearance/defaultCurrencySetting`

or

`/appearance/components/defaultCurrencySetting` wdyt?

Reply:
Thanks for the suggestions! I like `/components/appearance/...`. Then for other pages, let's say device settings it'll be `/components/devicesettings/...` . Looks good to me.

I'll make the change :)

Reply:
Also renamed `defaultCurrencySetting ` to be `defaultCurrencyDropdownSetting ` for consistency. :)